integration-tests/src/test/java/org/apache/druid/tests/indexer/ITRealtimeIndexTaskTest.java

```diff
@@ -51,6 +51,11 @@
   private static final Logger LOG = new Logger(ITRealtimeIndexTaskTest.class);
   private static final String REALTIME_TASK_RESOURCE = "/indexer/wikipedia_realtime_index_task.json";
   private static final String REALTIME_QUERIES_RESOURCE = "/indexer/wikipedia_realtime_index_queries.json";
+  /**
+   * The expected number of rows ingested for this test.
+   * The total number of rows of raw data is 22, but two rows will be rolled up into one row.
```

Comment:
Pardon my ignorance, where does the raw data come from?

Reply:
No worries. It's in the same class (https://github.com/apache/incubator-druid/blob/63c5d7add48d9fb6b301bf0baf9504a6b52267fc/integration-tests/src/test/java/org/apache/druid/tests/indexer/ITRealtimeIndexTaskTest.java#L85).